release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1196,6 +1196,16 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
 
 * Before this update, the remote write endpoint for a user workload monitoring Prometheus pods did not have custom CA bundles injected so that the endpoint could not be protected using SSL. With this update, the trusted CA bundle has been added to user workload monitoring Prometheus pods so that users can secure the remote write endpoint using SSL. (link:https://issues.redhat.com/browse/OCPBUGS-11958[*OCPBUGS-11958*])
 
+* With this release, as a security measure, Cross Origin Resource Sharing (CORS) headers are now disabled by default for Thanos Querier. If you still need to use CORS headers, you can enable them by setting the value of the `enableCORS` parameter to `true` for the `ThanosQuerierConfig` resource. (link:https://issues.redhat.com/browse/OCPBUGS-11889[*OCPBUGS-11889*])
```

Comment:
Shouldn't this be a new feature (assuming it isn't there already), where it is a change for better security? Might get better visibility, too.